### PR TITLE
Presets

### PR DESCRIPTION
**Pull request by skymakerjaime**
New awesome tutorial about presets (5th step is missing tho)

### CODE REVIEW

File: docs/tutorials-presets.md [hunk 1]
```diff
@@ -0,0 +1,279 @@
+---
+id: tutorials-presets
+title: Presets, Test & Debug
+sidebar_label: Presets, Test & Debug
+---
+_This tutorial takes approximately 10-20 minutes to complete. Good luck !_
+
+Here we cover the basics for creating projects with a development perspective. Following the _Test Driven Development_ method (or _TDD_), you will learn how to shield your complex and dynamic product against future changes.
```
Comment (axelSkyMaker) [inline docs/tutorials-presets.md]: Do we want to protect against future changes or bugs in future changes?

File: docs/tutorials-presets.md [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+- Presets for components
+- Presets for 2D & 3D representations
+- Tests & Debugging
+- Presets in drawings
+
+> We recommend you to check [My First App](my-first-app) before starting this tutorial.
+
+## 1. What is a preset?
+
+We call a _preset_ an specific case of your component, sketch or model, that helps you to visualize its behaviour. Separated from the component, it allows you to test and try different settings before you actually use it in your component and wherever this is used, e.g. in the app or in other components as a subcomponent.
```
Comment (axelSkyMaker) [inline docs/tutorials-presets.md]: Hard to understand last sentence

**Comment by axelSkyMaker** [comment]
Nice tutorial! Check my comments and see if you find them reasonable to fix.

File: docs/tutorials-presets.md (hunk 1 above)
Comment (skymakerjaime) [reply docs/tutorials-presets.md]: bugs in future changes sounds better

File: docs/tutorials-presets.md (hunk 2 above)
Comment (skymakerjaime) [reply docs/tutorials-presets.md]: "We call a _preset_ an specific case of your component, sketch or model, that helps you to visualize its behaviour. Separated from the component, it allows you to test and try different settings before you actually use it in your application." - sounds better?

Comment (axelSkyMaker) [reply docs/tutorials-presets.md]: Better!

**Pull request merged**